lib/fastlane/actions/carthage.rb

```diff
@@ -26,16 +25,21 @@ def self.description
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :command,
+                                       env_name: "FL_CARTHAGE_COMMAND",
```

Comment:
squashed: https://github.com/uny/fastlane/commit/09026ca8d8ecd6ac1000c6bce4e2af4a12e27444

thank you for your reviews.